src/webhooks/StaffWebhook.ts

```diff
@@ -25,5 +25,9 @@ export class StaffWebhook extends WebhookManager {
 		return process.env.STAFF_MESSAGE
 			? this.editMessage(process.env.STAFF_MESSAGE, { embeds: [embed] })
 			: this.send({ embeds: [embed] });
```

Comment:
Remove this, otherwise it will stop because of the return

Reply:
`embed.addField("Special Kid", "<@628829764135813160>");
		embed.addField("Supreme Leader", "<@511209271678074891>");
		return process.env.STAFF_MESSAGE
			? this.editMessage(process.env.STAFF_MESSAGE, { embeds: [embed] })
			: this.send({ embeds: [embed] });`
like this?

Reply:
ive updated it please check if thats what you meant. (i may or may not have put the code here because i forgot how)